install/kubernetes/values.yaml

```diff
@@ -171,6 +171,11 @@ tetragonOperator:
     # -- Enables the PodInfo CRD and the controller that reconciles PodInfo
     # custom resources.
     enabled: false
+  # -- Enables the Tetragon Operator metrics.
+  prometheus:
+    enabled: true
+    # -- The port at which to expose metrics.
+    port: 2113
```

Comment:
The port must be also passed to the operator in the [operator configmap](https://github.com/cilium/tetragon/blob/e287ce8957d5ed0439aa8a9d79df30b42629bc9b/install/kubernetes/templates/operator_configmap.yaml), as `metrics-bind-address`. Otherwise if you don't use the default, the k8s service won't match the port actually used by the operator pod.

Reply:
that's better though, however, we change the default port at `operator/cmd/serve/serve.go` believed that some change at there too that we can serve the port from config too? or am i missing something